### ISSUE TITLE
Color Picker laggy

### ISSUE THREAD

**Issue by dnnsjsk** [ThemeFuse/Brizy#1136]
Hey,

I find the colour picker to be quite laggy, it's hard to make fine adjustments. Same goes for typography controls. See the gif attached.

![giphy](https://user-images.githubusercontent.com/39319746/52798170-accf0280-3077-11e9-9092-91a3b7191a0c.gif)

**Comment by dnnsjsk** [comment]
did some more tests, it only really is that laggy when working on global colours. very smooth when working on module colours etc.

**Comment by thebrandingking** [comment]
Agreed, same thing here. Now I know Im not crazy or its only me. I find the entire platform slow and lagging. Using Firefox but will try other browsers to see. Obviously some kind of developmental problem.
